README.md

```diff
@@ -0,0 +1,38 @@
+# Operator for OSBuilder
+
+This repo has the operator which provides a K8S API for building [OSBuild](https://www.osbuild.org/) images.
+
+## Installation
+
+### Requirements
+
+- `osbuild >= 26`
```

Comment:
not sure if this is a direct requirement for building the operator.

Reply:
This is a requirement for building the osbuild-composer as mentioned [here](https://github.com/osbuild/osbuild-composer#requirements), so I thought it would be relevant here too. Should I remove it?

Reply:
Yes, please remove it. The operator uses published RPM, so it does not require any osbuild packages

Reply:
Done